src/Processing/ImportProcessingService.php

```diff
@@ -286,6 +260,51 @@ protected function processElement(string $configName, array $importDataRow, Reso
         }
     }
 
+    /**
+     * Process transformations for an element
+     * 
+     * @param ElementInterface $element
+     * @param array $importDataRow  
+     * @param MappingConfiguration[] $mapping
+     * @return MappingConfiguration|null Returns the mapping that was being processed.
```

Comment:
```suggestion
     * @param-out MappingConfiguration[]|null $mapping
```
looks like the right  way
https://phpstan.org/writing-php-code/phpdocs-basics#setting-parameter-type-passed-by-reference

Reply:
Added